pages/linux/alpine.md

```diff
@@ -1,15 +1,15 @@
 # alpine
 
-> A easy-to-use Email and Internet News program.
-> With pico/nano inspired interface and support for most modern Email services through IMAP.
+> An email client and Usenet newsgroup program with a pico/nano-inspired interface.
+> Supports most modern Email services through IMAP.
```

Comment:
"Email" is capitalised here? That doesn't feel right. it's only people / places that should be capitalised in the middle of a sentence, IIRC.

Reply:
That was an overlook -- I meant to lowercase it from the previous wording, but that one slipped. I'll change.